framework/index.scss

```diff
@@ -117,3 +117,4 @@
 @import 'utilities/ecl-padding';
 @import 'utilities/ecl-ratio';
 @import 'utilities/ecl-screen-reader';
+@import 'utilities/ecl-text';
```

Comment:
Was it included to make an example with the `--expired` modifier? I don't see any other reason that would be included in this ticket.